### PR TITLE
Use encrypted file for CLI deployment arguments

### PR DESCRIPTION
**Pull request by nre-ableton**
Ping @AbletonDevTools/gotham-city

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -40,12 +45,8 @@ devToolsProject.run(
   deployWhen: { return runTheBuilds.isPushTo(['master']) || env.FORCE_DEPLOY == 'true' },
   deploy: { data ->
     data['dtrImage'].push()
-    data['dtrImage'].deploy(
-      '8000',
-      '-v jenkins-nodes:/jenkins_nodes',
-      '--target-port 9100 --target-port 9323' +
-        " --url ${env.JENKINS_URL} /jenkins_nodes/output.json",
-    )
+    String cliArgs = encryptedFile.read(path: 'cliArgs.enc', credentialsId: CREDENTIAL_ID)
```
Comment (rco-ableton) [inline Jenkinsfile]: A bit petty, but I'm not a fan of `cliArgs.enc` as a filename—can we make it all lower case? Mixed case filenames seem weird to me.

**Review by rco-ableton — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (nre-ableton) [reply Jenkinsfile]: Fixed in d29591d.

File: Jenkinsfile [hunk 2]
```diff
@@ -1,6 +1,11 @@
 library 'ableton-utils@0.13'
 library 'groovylint@0.6'
 
+import groovy.transform.Field
+
+
+@Field static final String CREDENTIAL_ID = 'jenkins-node-scanner-password'
```
Comment (rco-ableton) [inline Jenkinsfile]: Would it make more sense to just use this string directly? Importing `Field` and creating a global seems like a lot of work just to save breaking a line (which is what I assume is going on on line 48—correct me if I'm wrong).

**Review by rco-ableton — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (nre-ableton) [reply Jenkinsfile]: Yes, the goal was to avoid a linebreak, but this also mirrors how we handle credentials in other projects.

Comment (nre-ableton) [reply Jenkinsfile]: Fixed in 2297e81.

File: Jenkinsfile (hunk 1 above)
Comment (nre-ableton) [reply Jenkinsfile]: BTW, I'm not a fan of `filenameswithoutdelimitersofanykindbecauseifindthemhardtoparse.txt`. 😄 So I went with the hyphenated `cli-args.enc`. Hope you're ok with this.

**Comment by nre-ableton** [comment]
@ablbot autosquash

File: Jenkinsfile (hunk 1 above)
Comment (rco-ableton) [reply Jenkinsfile]: Sure, it's fine by me 🙂

**Review by rco-ableton — APPROVED** [review]
LGTM 👍

**Pull request merged**